cmd/branchingconfigmanagers/schedule-generator/main.go

```diff
@@ -48,9 +50,29 @@ func validateOptions(o *options) error {
 	if err != nil {
 		return fmt.Errorf("invalid --log-level: %w", err)
 	}
+	if o.scheduleDir == "" {
+		return fmt.Errorf("--schedule-dir must be specified")
+	}
 	return o.kubernetesOptions.Validate(o.dryRun)
 }
 
+func validateSchedules(schedules ocplifecycle.Config) error {
+	var errs []error
+	for _, lifecyclePhaseByVersions := range schedules {
+		for version, lifecyclePhases := range lifecyclePhaseByVersions {
+			if _, err := ocplifecycle.ParseMajorMinor(version); err != nil {
```

Comment:
One thing that I'd like to have validated is that the events are sorted timewise (descending)